### PR TITLE
update apidoc

### PR DESCRIPTION
**Pull request by huixisheng**
update apidoc version

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "gulp-api-doc",
-  "version": "1.1.0",
+  "version": "1.1.1",
```
Comment (SamVerschueren) [inline package.json]: Do not update the version manually. I do it at release.

**Review by SamVerschueren — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (huixisheng) [reply package.json]: ok. I revert version update.

**Comment by SamVerschueren** [comment]
Released as `2.0.0`. Apparently that version of `apidoc` uses ES2015 syntax and thus didn't work on Node 0.10 and 0.12. So I removed support for those versions as well, hence the major version update.

**Comment by SamVerschueren** [comment]
And thanks for the pr :tada: :)!

**Pull request merged**